src/main/java/com/mide/gangsaeng/board/Board.java

```diff
@@ -0,0 +1,8 @@
+package com.mide.gangsaeng.board;
+
+public class Board {
```

Comment:
Change to Immutable?

Reply:
Yes, VO (value object) is Read-Only , so make immutable is better choice.